src/BootstrapAdminUi/templates/shared/helper/table.html.twig

```diff
@@ -26,11 +26,15 @@
 
     <tr class="item" {{ sylius_test_html_attribute('row') }} {{ sylius_test_html_attribute('resource-id', row.id) }}>
         {% if definition.actionGroups.bulk is defined and definition.getEnabledActions('bulk')|length > 0 %}
-            <td class="center aligned"><input data-check-all-group="index" class="form-check-input" type="checkbox" value="{{ row.id }}" /></td>
+            <td class="text-center"><input data-check-all-group="index" class="form-check-input" type="checkbox" value="{{ row.id }}" /></td>
         {% endif %}
         {% for field in definition.enabledFields|sylius_sort_by('position') %}
             <td{% if field.options.vars.td_class is defined %} class="{{ field.options.vars.td_class }}"{% endif %}>
-                {{ sylius_grid_render_field(grid, field, row) }}
+                {% if field.name == 'code' %}
+                    <code>{{ sylius_grid_render_field(grid, field, row) }}</code>
```

Comment:
if it was regular twig render, then styling of the field should have been done via theme, but guess it's not applicable over here.

Reply:
Indeed, interesting comment. For now, I prefer keeping admin-bundle and bootstrap-admin-ui synced.